config/spells/ability.json

```diff
@@ -476,4 +476,193 @@
 			}
 		}
 	},
+	"firstAid" : {
+		"targetType" : "CREATURE",
+		"type": "ability",
+		"name": "First Aid",
+		"school" : {},
+		"level": 1,
+		"power": 10,
+		"defaultGainChance": 0,
+		"gainChance": {},
+		"animation":{
+			"affect":["SP12_"]
+		},
+
+		"sounds": {
+			"cast": "REGENER"
+		},
+		"levels" : {
+			"base":{
+				"description" : "",
+				"aiValue" : 0,
+				"power" : 10,
+				"cost" : 0,
+				"targetModifier":{"smart":true},
+				"battleEffects":{
+					"heal":{
+						"type":"core:heal",
+						"healLevel":"heal",
+						"healPower":"permanent",
+						"optional":true
+					}
+				},
+				"range" : "0"
+			},
+			"none" :{
+				"power" : 10
+			},
+			"basic" :{
+				"power" : 50
+			},
+			"advanced" :{
+				"power" : 75
+			},
+			"expert" :{
+				"power" : 100
+			}
+		},
+		"flags" : {
+			"positive": true
+		},
+		"targetCondition" : {
+			"nonMagical" : true,
+			"noneOf" : {
+				"bonus.SIEGE_WEAPON" : "absolute" //Is war machines affected by first aid tent?
+			}
+		}
+	},
+	"catapultShot" : {
```

Comment:
Can you confirm that all catapult "spells" work like before with battle UI?
Meaning, wall targets are selectable, selecting a target will fire shot as before (and twice in case of expert ballistics / cyclops king)

Reply:
@IvanSavenko I checked catapult - it works as before with all ballistics levels. Only cursed ground remains for checking, but I do not know any map with towns on cursed ground. If will found - will check too. Both with First Aid.

Reply:
Here you go: https://github.com/vcmi/vcmi/issues/1589
> Vanilla map "Freedom"

However cursed ground should not affect abilities so likely fine.

Reply:
@IvanSavenko this is only one visual issue I can see - turret shooter removed from turret after first shot even turret itself is killed by second shot. It was be like this in develop before?

Reply:
Expert ballistics in question - so it is only visual issue.

Reply:
@IvanSavenko works on cursed ground.

Reply:
> turret shooter removed from turret after first shot even turret itself is killed by second shot. It was be like this in develop before

Yes, remember something like this from develop

Reply:
@IvanSavenko First Aid generates this too. Is this should be removed or it is normal?

Reply:
Should be removed. H3 does not have that message & it is not intuitive / something that player expect

Reply:
@IvanSavenko strings removed. But I think we should have some strings about catapult shot in battle log. For First Aid - it is ok even without string.